packages/application/src/services.tsx

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2018-2020 IBM Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Dialog, showDialog } from '@jupyterlab/apputils';
+import * as React from 'react';
+
+import { RequestHandler } from './requests';
+
+/**
+ * A utilities class for various elyra services.
+ */
+export class FrontendServices {
```

Comment:
Jupyter already has `JupyterFrontEnd`  class which I really think that it wasn't a good name choice, and with this new one, with a similar name, could confuse users? What about just calling this like `Application` and then we would have `Application.getMetadata()`?

Reply:
I also am not a fan our this name, but I don't think Application is any better. I's be more in favor of just naming it `MetadataService` since that's what it is.